samples/RT-sample-app/nodejs/README.md

```diff
@@ -0,0 +1,149 @@
+---
+page_type: sample
+products:
+- office-365
+languages:
+- javascript
+title: Specify relationships between app capabilities
+description: Microsoft Teams RT sample app in Node.js which specify relationships between app capabilities using (elementRelationshipSet)
+extensions:
+  contentType: samples
+  createdDate: 10/19/2024 10:02:21 PM
+urlFragment: officedev-microsoft-teams-samples-RT-sample-app-nodejs
+---
+
+# Microsoft Teams RT Sample App Feature.
```

Comment:
We should use "Microsoft 365" instead of "Teams" as RT is for M365 platform, not just for Teams.